packages/web-components/fast-foundation/src/progress/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from "./base-progress";
-export * from "./progress/index";
-export * from "./progress-ring/index";
+export * from "./progress.template";
+export * from "../progress-ring/index";
```

Comment:
Same question here as with tabs: 

I know these were previously being exported from this file, but now that we've gone to a flat structure, does this still make sense? I would expect that we might just export these from their own folders and export them directly from the root.